common/lib/xmodule/xmodule/x_module.py

```diff
@@ -593,7 +593,6 @@ def bind_for_student(self, xmodule_runtime, user_id, wrappers=None):
                 # not the most elegant way of doing this, but if we're removing
                 # a field from the module's field_data_cache, we should also
                 # remove it from its _dirty_fields
-                # pylint: disable=protected-access
```

Comment:
I think this might be superfluous because there's a similar pragma three lines above?